capstone/src/main/webapp/youtubeScript.js

```diff
@@ -21,13 +21,13 @@ const ATTRIBUTES_BY_LANGUAGE = {
   'pt': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT']
 };
 
+/** Category names are mapped to youtube category numbers */
 const YOUTUBE_CATEGORIES = {
   'Autos&Vehicles': 2,
   'Comedy': 23,
   'Entertainment': 24,
   'Film&Animation': 1,
   'Gaming': 20,
-  'How-to&Style' : 26,
```

Comment:
Just curious, any reason why this was removed?

Reply:
I was experiencing issues where results were not consistently returned. I consulted online forums and found others were having similar issues. I plan to continue experimenting to see if issue is resolved.